api/controllers/definition_manager/information-object.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * definition_manager/information-object.js
+ *
+ *
+ * url:     get / /definition/info/object/:ID
+ * header:  X-CSRF-Token : [token]
+ * params:
+ */
```

Comment:
Can you add [apidoc](https://github.com/apidoc/apidoc) comments and update the README (`npm run doc`)?